ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/TheirDetailsMapperTest.java

```diff
@@ -42,10 +43,11 @@ public void shouldMapCompanyToCCD() {
         TheirDetails party = SampleTheirDetails.builder().companyDetails();
 
         //when
-        CCDDefendant ccdParty = defendantMapper.to(party, null, null);
+        CCDDefendant.CCDDefendantBuilder builder = CCDDefendant.builder();
```

Comment:
instead of repeating this line in every single test case , you could have just added a setup method like below

private CCDDefendant.CCDDefendantBuilder ccdDefendantBuilder;
    
    @Before
    public void setup(){
        ccdDefendantBuilder = CCDDefendant.builder();
    }